docs/extensions/bundles.md

```diff
@@ -5,7 +5,8 @@ readTime: 5 min read
 
 # Extension Bundles
 
-> Extension bundles allow you to combine and share dependencies between one or more extensions.
+> Extension bundles allow you to combine and share dependencies between one or more extensions. They are developed using
+> JavaScript / Node.js.
```

Comment:
We should add an info about when the usage of a bundle makes sense (besides sharing dependencies), something like: a bundle can be used when an extension consists of a combination of several related sub-extensions

Reply:
Just noticed it also (and especially) needs to be updated over here: https://github.com/directus/directus/pull/19727/files#diff-9fa281b4b167fa082467a611ec865bfba6f1e9d880bde5a99fb891aa01986f44R87-R89